class-5/class.js

```diff
@@ -0,0 +1,13 @@
+class Car {
+  constructor(brand) {
+    this.brand = brand;
+  }
+
+  strartEngine() {
```

Comment:
I think you have misspelled this function name, did you mean it **`startEngine`**?

Reply:
Yes it is **startEngine**, I am solving it .